index.js

```diff
@@ -15,20 +20,39 @@ const prettySyntaxError = (err) => {
   }
 };
 
+const warnIf = (condition, warning) => {
+  if (condition) {
+    logger.warn(`${warning}`);
+  }
+};
+
 class BabelCompiler {
   constructor(config) {
     if (!config) config = {};
     const options = config.plugins &&
       (config.plugins.babel || config.plugins.ES6to5) || {};
+    if (options && !config.plugins.babel) {
+      warnIf(config.plugins.ES6to5, warns.ES6to5);
+    }
+
     const opts = Object.keys(options).reduce((obj, key) => {
       if (key !== 'sourceMap' && key !== 'ignore') {
         obj[key] = options[key];
       }
       return obj;
     }, {});
+
     opts.sourceMap = !!config.sourceMaps;
-    if (!opts.presets) opts.presets = ['es2015', 'es2016'];
+    if (!opts.presets || opts.presets.length === 0) {
+      if (!config.lookupConfigs) {
```

Comment:
Let's check `babelrc` `package.json`'s `babel` field and lastly, `dependencies` and `devDependencies` of `package.json`